cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -61,11 +61,13 @@ def build_load_config(self) -> RarediseaseLoadConfig:
         self.include_case_files(load_config)
         self.get_sample_information(load_config)
         self.include_pedigree_picture(load_config)
-        load_config.custom_images = self.load_custom_image_sample()
+        custom_images = self.load_custom_image_sample()
+        if custom_images:
+            load_config.custom_images = custom_images
```

Comment:
I think that this if statement will always be True if the case has samples.

Reply:
Indeed, reverting those and will address the issue for raredisease in another PR